docs/extend/plugin_api.md

```diff
@@ -14,9 +14,13 @@ weight=1
 Docker plugins are out-of-process extensions which add capabilities to the
 Docker Engine.
 
+This document describes the Docker Engine plugin API generally available in
+Docker Engine 1.12 and earlier. To view information on plugins managed by Docker
+Engine, refer to [Docker Engine plugin system](plugins.md).
```

Comment:
@thaJeztah comment SGTM. But I would say 1.12 experimental build here if you want to publish the doc before we release 1.13 . Then we could update to 1.13 for the GA release.

We need to come up with a style for talking about experimental features. Actually we should probably add: `advisory=expirmental` to the header and udpate https://github.com/docker/docs-base/blob/master/data/advisories.toml accordingly